### PR TITLE
add when condition to decide where to run a check

### PR DESCRIPTION
**Pull request by angelabriel**
and
fix a typo in expectation section of check 49591F.yaml

### CODE REVIEW

File: priv/catalog/0B6DB2.yaml [hunk 1]
```diff
@@ -48,14 +48,15 @@ remediation: |
     - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-storage-protect.html
     - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-1.11.5.5.4.4
 
+when: env.provider == "azure" || env.provider == "nutanix" || env.provider == "kvm"
```
Comment (arbulu89) [inline priv/catalog/0B6DB2.yaml:51]: Maybe `env.provider in ["azure", "nutanix", "kvm"]` looks bettern ans shorter?
What do you think?

Comment (angelabriel) [reply priv/catalog/0B6DB2.yaml:51]: yes, looks nice. I wasn't aware that I can use 'all' rhai expressions inside of the `when` clause
Will fix it soon, thanks

**Review by arbulu89 — APPROVED** [review]
Thanks @angelabriel

**Review by nelsonkopliku — APPROVED** [review]
Awesome! Thanks!

**Review by dottorblaster — APPROVED** [review]
LGTM

**Pull request merged**